src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -124,7 +124,7 @@ internal static void ParseResponseObject(IDictionary<string, object> responseObj
                 object statusValue;
                 if ((responseObject.TryGetValue("statusCode", out statusValue, ignoreCase: true) ||
                      responseObject.TryGetValue("status", out statusValue, ignoreCase: true)) &&
-                     (statusValue is int || statusValue is string))
+                     (statusValue is long || statusValue is int || statusValue is string))
```

Comment:
For completeness we should probably handle all the integral types, e.g. int, long, ulong, short, ushort, ...

Recommend adding an `internal static bool TryParseStatusCode(object value)` method that is used here. Internally it can then do all the right checks for the rest of the integral types. Also please add a unit test to HttpBindingTests.cs for the new helper. Can be a [Theory] test that tests the various inputs, e.g. like [this test](https://github.com/Azure/azure-webjobs-sdk-script/blob/da6e7ed5d606bd7a2c70e2b3f262220852abfd39/test/WebJobs.Script.Tests/FeatureFlagsTests.cs#L17).